src/estimation/msm_criterion.py

```diff
@@ -312,9 +326,24 @@ def _calculate_share_known_cases(sim_out):
     return avg_share_known
 
 
-def _get_empirical_moments(df, age_group_sizes, state_sizes):
+def _calculate_period_virus_share(df):
+    df = df[["virus_strain", "date", "newly_infected"]]
```

Comment:
Since we add the "b117" column, I think we need a `copy` here :thinking: 

```suggestion
    df = df.copy(deep=True)
    df = df[["virus_strain", "date", "newly_infected"]]
```

Reply:
I think taking out two columns makes a shallow copy, but I'm not sure, so we should make one explicitly. Deep copy is too expensive here!